src/elements/tokens/TokenTable.tsx

```diff
@@ -23,12 +23,17 @@ export const TokenTable = ({ searchInput, setSearchInput }: Props) => {
   const tokens = useAppSelector(getTokenTableData);
 
   const data = useMemo<Token[]>(() => {
-    return tokens.filter(
-      (t) =>
+    return tokens.filter((t) => {
```

Comment:
It's wasteful to call `searchInput.toLowerCase()` so many times (although maybe the engine optimizes this).
In any case, I'd do
`const search = searchInput.toLowerCase();`
and use it instead.

Reply:
done